### ISSUE TITLE
  Формирование Журнала-ордера по дате проводки и по дате документа

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#3083]
Originally reported on Google Code with ID 3083

```

  Как сформировать Журнал-ордер по дате проводки и по дате документа и при этом, чтобы
столбцы дат корректно назывались ?
  Сейчас уже исправлена аналитика "Дата" на "Дата проводки". Теперь столбец "Дата"
отображается, как "Дата проводки" - всё верно. Необходимо сформировать Журнла-ордер
не только по дате проводки, но и по дате документа. Попробуем воспользоваться галкой
"Расширенное отображение полей". При этом в таблице GD_DOCUMENT в поле "Поля для расширенного
отображения (на английском через запятую)" пропишем DOCUMENTDATE. В результате получим
два столбца дат: "Дата проводки" и "Дата". В данном случае столбец "Дата" должен называться
"Дата документа". Попробуйте сформировать Журнал-ордер по счёту 97.01 и всё станет
ясно.
  Можно в таблице GD_DOCUMENT в поле DOCUMENDATE переименовать локализованное наименование
в "Дата документа". Но из-за этого, возможно, во всех документах переименуется поле
"Дата" в "Дата документа". Но, будет ли в Журнале-ордере столбец "Дата" называться
"Дата документа" ? При попытке такого переименования ничего не меняется.

```

Reported by `NikolayUkleyko` on 2013-04-30 14:23:12

<hr>
- _Attachment: Таблица GD_DOCUMENT и поле DOCUMENTDATE.png<br>![Таблица GD_DOCUMENT и поле DOCUMENTDATE.png](https://storage.googleapis.com/google-code-attachments/gedemin/issue-3083/comment-0/Таблица GD_DOCUMENT и поле DOCUMENTDATE.png)_

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2013-04-30 14:24:08
- **Labels added**: Type-Defect, Priority-Medium, Accountancy, GedeminEXE, Component-UI

**Comment by gsbelarus** [comment]
```

  Скажите кратко: можно или нельзя ?
  У кого какие предложения по данной теме ?

```

Reported by `NikolayUkleyko` on 2013-10-24 13:22:42

<hr>
- _Attachment: В Журнале-ордере переименовать с'Дата' на 'Дата документа'.png<br>![В Журнале-ордере переименовать с'Дата' на 'Дата документа'.png](https://storage.googleapis.com/google-code-attachments/gedemin/issue-3083/comment-2/В Журнале-ордере переименовать с'Дата' на 'Дата документа'.png)_

**Comment by gsbelarus** [comment]
```
Я за то что бы такая комбинация вообще была запрещена при построении журнала ордера.
потому что в этом случае не возможно правильно отоборазить колонку сальдо.
```

Reported by `mikleshoihet` on 2013-10-24 13:27:34

**Comment by gsbelarus** [comment]
```

  Данная комбинация имеет место быть при работе с 97-м счетом. Так удобнее выявлять
ошибки.

```

Reported by `NikolayUkleyko` on 2013-10-24 14:44:10

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2013-11-21 06:58:13
- **Status changed**: `Duplicate`
- **Merged into**: #3269
